src/index.js

```diff
@@ -192,6 +192,15 @@ class Libp2p extends EventEmitter {
       this.pubsub = PubsubAdapter(Pubsub, this, this._config.pubsub)
     }
 
+    // Create rendezvous if provided
+    if (this._modules.rendezvous) {
+      const Rendezvous = this._modules.rendezvous
+      this.rendezvous = new Rendezvous({
```

Comment:
Considering that we will provide the `libp2p.discovery` API, I am thinking wether we should provide `libp2p.rendezvous`, or just leave it as `libp2p._rendezvous`. Thoughts?
If we keep this as it is, this PR should include API docs for rendezvous